_i18n/en.yml

```diff
@@ -530,7 +530,8 @@ faq:
     Monero fixes this problem by being ASIC-resistant: it uses an algorithm (@randomx) that strongly reduces the efficiency of ASICs, making them not profitable to build. Miners can use common consumer hardware, which allows them to compete fairly. The Monero network is currently protected by thousands of miners using 'regular' computers. This results in a network much harder to attack, no miner having significant advantage over other miners (they all use more or less the same hardware).
   qmaxsupply: What is Monero's maximum supply?
   amaxsupply: Monero has a fixed emission rate, not a set maximum supply. Around May 2022, Monero's emission will drop to and permanently remain at 0.3 XMR per minute (0.6 XMR per @block). This is approximately 1% inflation for the first year and will approach 0% inflation in future years. This @tail-emission allows for permanent incentives to secure Monero, even in the far future, while keeping inflation at a very low percent.
-
+  qtailemission: Why does Monero not have a maximum supply?
+  atailemission: Miners process transactions on the Monero network by @mining blocks. The miner of a @block is paid the constant block reward of .6 XMR, and the @transaction fees of the users who have transactions in that block. Monero has the block reward rather than relying solely on the transaction fees to give the miners incentive to keep securing the network with their hashrate, and keep transaction fees low. The @tail-emission caused by this constant block reward creates an inflation rate of less than 1% which trends towards 0% over time. The fixed emission of the currency ensures human corruption cannot over inflate the supply. Keeping the network predictable, decentralized, and secure.
```

Comment:
This is too much text in one variable, please split it in two or even three.

Reply:
Could you confirm if you mean 'split it into paragraphs' or 'spit the 1 paragraph up until multiple strings to help translators' (i am going to make a helper script for adding a faq some time)

Reply:
I have split up the answer into two paragraphs in the most recent commit.